litellm/llms/sagemaker.py

```diff
@@ -9,6 +9,9 @@
 import sys
 from copy import deepcopy
 import httpx  # type: ignore
+import boto3
+import aioboto3
```

Comment:
this would make boto3 a default import for litellm

can we move it into the functions so it's only imported for sagemaker

Reply:
Went into more detail on the other comment- this was needed in order to make the file testable from the perspective of figuring out which regions were being used.

What is the impact of having the libraries imported at the greater LiteLLM level? They won't make network calls unless the clients are being invoked, which will only happen within the SageMaker file here. Otherwise inside the function, we have to import the library every single time on every SageMaker-related invocation if I'm reading that correctly

Reply:
my understanding is this would raise errors if i don't have this in my env 

since this isn't a dep of the library, it would be incorrect to require a user to pip install boto3 just to use litellm

Reply:
Ah I see, that makes sense- yes that would be true, but also true of any other provider, correct?
I will revise, thanks for that context- I didn't factor in _unwanted_ dependencies.